src/Components/Game/Game.js

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import './Game.css';
+import { 
+  Container,
+  Row,
+  Col,
+} from 'react-bootstrap';
+import { CountdownTimer } from '../Timer/Timer';
+
+import person from '../../assets/icons/Icon material-person.png';
+import person2x from '../../assets/icons/Icon material-person@2x.png';
+import gamepad from '../../assets/icons/Icon awesome-gamepad.png';
+import gamepad2x from '../../assets/icons/Icon awesome-gamepad@2x.png';
+import replay from '../../assets/icons/Icon open-reload.png';
+import replay2x from '../../assets/icons/Icon open-reload@2x.png';
+import cross from '../../assets/icons/Icon metro-cross.png';
+import cross2x from '../../assets/icons/Icon metro-cross@2x.png';
+
+function CharList(props) {
+  const listitems = props.word.split('').map((char, index) => {
+    return (
+      <span key={index}
+        style={{
+          color: (props.currentAnswer[index] ? 
+            props.currentAnswer[index].toLowerCase() === char.toLowerCase() ? '#54ba18' : '#445298' : '#ffffff')
+        }}>
+        {char}
+      </span>
+    );
+  });
+  return (
+    <div>{listitems}</div>
+  );
+}
+
+export class Game extends React.Component{
+
+  EndGame = (props) => {
```

Comment:
EndGame seems to be a separate component. do not put components as local variables in your class.
One file should have only one component.

variable naming is also incorrect. local variables should always follow camelCase syntax.